src/style.css

```diff
@@ -1,3 +1,73 @@
-body {
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1415px;
```

Comment:
all numbers should be divisible by 4 or 8

Reply:
hope 1416 suits, it is divisible by 4.